components/Common/DarkModeToggle/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ModeNightIcon from '@mui/icons-material/ModeNight';
+import BrightnessMediumIcon from '@mui/icons-material/BrightnessMedium';
+import styles from './index.module.scss';
+import { useTheme } from './useTheme';
+
+const DarkModeToggle = () => {
+  const [theme, toggleTheme] = useTheme();
+
+  const isDark = theme === 'dark';
+
+  return (
+    <button
+      type="button"
+      className={styles.darkModeToggle}
+      onClick={() => {
```

Comment:
No need of `{}` here. Can simply be `onClick={() => toggleTheme()}`. Also, any reason why the `?.` Can the function be somehow null?

Reply:
> Also, any reason why the ?. Can the function be somehow null?

Yeah , the ts is throwing error, function could be undefined because of default context value